server/internal/platform/pg/organization.go

```diff
@@ -2,6 +2,7 @@
 package pg
 
 // Organization is a struct type for organization entities.
+// TODO: REMOVE STRUCT FROM HERE, AND DEFINE IN SEPERATE FILE IN FLOWTHRU FOLDER
```

Comment:
Try to make this change or remove the comment.

Reply:
I'll remove the comment for now. I'll leave this as something to do once the appropriate endpoint is being worked on.